sdk/azurestackhci/Azure.ResourceManager.Hci/src/Generated/Models/HciClusterNode.cs

```diff
@@ -46,6 +52,10 @@ internal HciClusterNode(string name, float? id, WindowsServerSubscription? windo
         public float? Id { get; }
         /// <summary> State of Windows Server Subscription. </summary>
         public WindowsServerSubscription? WindowsServerSubscription { get; }
+        /// <summary> Type of the cluster node hardware. </summary>
+        public ClusterNodeType? NodeType { get; }
+        /// <summary> Edge Hardware Center Resource Id. </summary>
+        public string EhcResourceId { get; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#d4cebe383870410cab2e14d0709f78bb).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#9f6fc0ae7c8e497e98d7979dadf97a4c).

Reply:
This is string and not any Guid or ResourceIdentifier.
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2022-12-01/clusters.json